bioscrape/sbmlutil.py

```diff
@@ -65,7 +65,11 @@ def import_sbml(sbml_file, bioscrape_model = None, input_printout = False, **kwa
     for reaction in model.getListOfReactions():
         # get the propensity
         kl = reaction.getKineticLaw()
+
+
         # capture any local parameters
+        # also must save renamed local parameters to rename annotations later
+        renamed_params = {}
```

Comment:
Seems like the changes from the previous PR are not merged/in sync yet? Why are the changes in sbmlutil on removing underscores from parameters showing up here?

Reply:
Seems like you created this PR to master, maybe that's the reason?

Reply:
That was just a mistake - changed the merge into dev instead. Everything look okay?